### DISCUSSION TITLE
Federated Learning Loss not logged correctly?

### DISCUSSION THREAD

**Discussion by LSnyd** [Project-MONAI/tutorials#446] [General]
Hi there, 

I am currently trying to get a Federated Learning training running according to the Spleen Example.

I got the training process running and get a somehow meaningful output. However, when I am checking my loss on tensorboard (from fl_workspace/server/run_1/mmar_client1/custom/models)  

I am receiving the following output: 

![image](https://user-images.githubusercontent.com/38072998/142003735-57222156-5be5-433e-9895-8ed1ec626b44.png)

It seems like the loss is not correctly logged over the 200 global epochs ( 5 local epochs). Could anyone help me out with that?

**Comment by LSnyd** [comment]
Hi @yiheng-wang-nv, 
as you were tagged in my other FL problem, I am assuming that you are the FL expert :) 
Could you maybe also help me out with this problem?

**Comment by LSnyd** [comment]
I just found out that I am getting this result, since multiple tensorboard logfiles are included and overlapped in this plot. It looks fine, when I am only keeping the log results of one run.

**Reply by yiheng-wang-nv** [reply]
Hi @LSnyd , sorry for late reply, for this question do you still have other questions?
